README.md

```diff
@@ -135,15 +135,26 @@ resolve
 Currently it supports the following options :
 
 ``` clj
-{:output-deps-edn-file "deps.edn"
- :input-deps-edn-file "deps.edn"
- :versions-edn-file ".deps-versions.edn"
- :modules-dir "modules"
+{:versions-file "deps.edn"
+ :versions-keypath `[:exoscale.deps/managed-dependencies]`
+ :modules-patterns-keypath `[:exoscale.deps/modules-patterns]`
+ :modules-patterns ["modules/*/deps.edn"]
  :dry-run? false}
 ```
 
+- `dry-run?`: When set to `true`, will only print the intended changes
+- `versions-file`: File containing the managed dependencies configuration
+- `versions-keypath`: Key path at which the managed dependencies configuration is to be found in
+   the configured `versions-file`. Defaults to `[:exoscale.deps/managed-dependencies]`
+- `project-files-patterns-keypath`: Key path at which the globbing pattern for project files is to be found
```

Comment:
Even though we stick things in a `modules/` directory most of the time. We are actually looking for project files, or deps files? I tried to find a naming to reflect that.